contract/package.json

```diff
@@ -11,26 +11,32 @@
     "docker:make": "docker compose exec agd make -C /workspace/contract",
     "make:help": "make list",
     "start": "yarn docker:make clean start-contract print-key",
-    "build": "exit 0",
+    "build": "node_modules/agoric/bin/agoric run scripts/build-game1-start.js",
```

Comment:
is `node_modules/agoric/bin/agoric` necessary? Doesn't yarn arrange for `agoric` to work?

Reply:
The scope of this PR seems to be expanding :)

Do you plan to keep the commits separate or squash?